tests/x11/shutdown.pm

```diff
@@ -1,48 +0,0 @@
-# SUSE's openQA tests
-#
-# Copyright © 2009-2013 Bernhard M. Wiedemann
-# Copyright © 2012-2017 SUSE LLC
-#
-# Copying and distribution of this file, with or without modification,
-# are permitted in any medium without royalty provided the copyright
-# notice and this notice are preserved.  This file is offered as-is,
-# without any warranty.
-
-# Summary: Initiate a system shutdown, taking care of differences between the desktops
-#   don't use x11test, the end of this is not a desktop
-# Maintainer: Dominique Leuenberger <dimstar@opensuse.org>
```

Comment:
@DimStar77 because this PR removes the module that you "maintained", please share your thoughts on this PR

Reply:
Certainly no objection against 'my module' (turns out the two maintainer lines in this file are the only ones written by me :) )